Kommunicate/Classes/KMPushNotificationHandler.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  KMPushNotificationHandler.swift
+//  Kommunicate
+//
+//  Created by Shivam Pokhriyal on 23/08/19.
+//
+
+import Foundation
+import Applozic
+import ApplozicSwift
+
+public class KMPushNotificationHandler: Localizable {
+    public static let shared = KMPushNotificationHandler()
+    var navVC: UINavigationController?
+
+    var contactId: String?
+    var groupId: NSNumber?
+    var conversationId: NSNumber?
+    var configuration: KMConfiguration!
+
+    /// Make it true to hide chat list on press of notification
+    public static var hideChatListOnNotification: Bool = false
```

Comment:
This should be true by default otherwise for the old users existing behavior will change.